api/libproject/devstart.rb

```diff
@@ -252,38 +248,37 @@ def create_db_creds(args)
   end
 
   instance_name = "#{project}:us-central1:workbenchmaindb"
-  creds_file = Tempfile.new("#{project}-vars.env")
-  if creds_file
+  db_creds_file = Tempfile.new("#{project}-vars.env")
+  if db_creds_file
     begin
-      creds_file.puts "DB_CONNECTION_STRING=jdbc:google:mysql://#{instance_name}/workbench"
-      creds_file.puts "DB_DRIVER=com.mysql.jdbc.GoogleDriver"
-      creds_file.puts "DB_HOST=127.0.0.1"
-      creds_file.puts "DB_NAME=workbench"
-      creds_file.puts "CLOUD_SQL_INSTANCE=#{instance_name}"
-      creds_file.puts "LIQUIBASE_DB_USER=liquibase"
-      creds_file.puts "LIQUIBASE_DB_PASSWORD=#{workbench_password}"
-      creds_file.puts "MYSQL_ROOT_PASSWORD=#{root_password}"
-      creds_file.puts "WORKBENCH_DB_USER=workbench"
-      creds_file.puts "WORKBENCH_DB_PASSWORD=#{workbench_password}"
-      creds_file.close
-
-      # Get service account credentials
-      service_account_creds_file = Tempfile.new("#{project}-creds.json")
-      get_service_account_creds_file(project, account, service_account_creds_file)
-      begin
-        activate_service_account(service_account_creds_file)
-        copy_file_to_gcs(creds_file.path, "#{project}-credentials", "vars.env")
-      ensure
-        delete_service_accounts_creds(project, account, service_account_creds_file)
-      end
+      db_creds_file.puts "DB_CONNECTION_STRING=jdbc:google:mysql://#{instance_name}/workbench"
+      db_creds_file.puts "DB_DRIVER=com.mysql.jdbc.GoogleDriver"
+      db_creds_file.puts "DB_HOST=127.0.0.1"
+      db_creds_file.puts "DB_NAME=workbench"
+      db_creds_file.puts "CLOUD_SQL_INSTANCE=#{instance_name}"
+      db_creds_file.puts "LIQUIBASE_DB_USER=liquibase"
+      db_creds_file.puts "LIQUIBASE_DB_PASSWORD=#{workbench_password}"
+      db_creds_file.puts "MYSQL_ROOT_PASSWORD=#{root_password}"
+      db_creds_file.puts "WORKBENCH_DB_USER=workbench"
+      db_creds_file.puts "WORKBENCH_DB_PASSWORD=#{workbench_password}"
+      db_creds_file.close
+
+      activate_service_account(creds_file)
+      copy_file_to_gcs(db_creds_file.path, "#{project}-credentials", "vars.env")
     ensure
-      creds_file.unlink
+      db_creds_file.unlink
     end
   else
     raise("Error creating file.")
   end
 end
 
+def create_db_creds(args)
+  run_with_creds(args, Proc.new { |project, account, creds_file|
+    do_create_db_creds(project, account, creds_file)
+  })
+end
+
```

Comment:
To clarify, you could remove `connect-to-cloud-db` and `run-cloud-migrations` as registered commands (below here) while still having Circle use those functions (as is done in the UI script). That let's us reuse common functionality and punt on the "how should developers run these operations" question.